packages/jumpstarter-driver-corellium/jumpstarter_driver_corellium/corellium/api.py

```
@@ -0,0 +1,170 @@
+        data = {
+            'apiToken': self.token
+        }
+
+        try:
+            res = requests.post(f'{self.baseurl}/v1/auth/login', json=data)
+            res.raise_for_status()
+            data = res.json()
+        # except (requests.exceptions.RequestException, requests.exceptions.HTTPError) as e:
+        # except requests.exceptions.HTTPError as e:
+        except Exception as e:
+            raise CorelliumApiException(str(e))
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)